packages/core/src/components/Button/Button.scss

```diff
@@ -185,3 +185,119 @@ Button inline with field
 .ds-c-button > svg {
   @include inline-icon;
 }
+
+/* Show focus styles in demo. */
+.ds-c-button--focus-1 {
```

Comment:
Personally I think the plain box focus style works. In other sites I have seen an even larger outline width (like 4px) but idk how others feel about that. Also I'm curious to know what other colors would look like... Are there other options that have good contrast?

Reply:
> Also I'm curious to know what other colors would look like... Are there other options that have good contrast?

Yeah, we can definitely try other colors. None of our other secondary blues meet the proper color contrast. We could use some of our primary colors, but most would be even darker than the existing one.